src/app/api/meme.service.ts

```diff
@@ -95,7 +110,7 @@ export class MemeService {
    * @param {number} memeId
    */
   async getMemeDetails(memeId: number): Promise<Meme> {
-    if (this.memes[memeId]) {
+    if (this.memes[memeId] && (this.memes[memeId].myVote !== undefined)) {
```

Comment:
You can remove this, this bug was due to the user's vote status not being returned for lists of memes.

Fixed in https://github.com/MemePlace/Server/pull/63